spec/unit/lib/temporal_spec.rb

```diff
@@ -1,16 +1,17 @@
 require 'temporal'
 require 'temporal/workflow'
+require 'pry'
```

Comment:
Happy to include it as a spec dependency, but not in a particular spec file

Reply:
Oops, meant to remove this. I may stick it spec_helper though.

Reply:
Done.